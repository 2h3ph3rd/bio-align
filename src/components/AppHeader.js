import * as React from 'react'
import Box from '@mui/material/Box'

export default function AppHeader() {
  return (
    <Box sx={style.container}>
      <img src={process.env.PUBLIC_URL + '/header.png'} alt="header" />
    </Box>
  )
}

const style = {
  container: {
    textAlign: 'center',
    mb: 1,
  },
}
