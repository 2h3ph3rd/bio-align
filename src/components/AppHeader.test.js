import React from 'react'
import renderer from 'react-test-renderer'

import AppHeader from './AppHeader'

it('renders correctly', () => {
  const tree = renderer.create(<AppHeader></AppHeader>).toJSON()
  expect(tree).toMatchSnapshot()
})
