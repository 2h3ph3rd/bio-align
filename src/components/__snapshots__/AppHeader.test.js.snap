// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation4 css-13qyio5-MuiPaper-root"
>
  <img
    alt="header"
    src="/header.png"
    style={
      Object {
        "height": "100%",
        "objectFit": "contain",
        "width": "100%",
      }
    }
  />
</div>
`;
